README.md

```diff
@@ -529,6 +529,49 @@ server.listen(443, () => {
 
 </details>
 
+<details>
+<summary>Server usage with <a href="https://github.com/apollographql/apollo-server/tree/main/packages/apollo-server-express">Apollo Server Express</a></summary>
+
+```typescript
+import https from 'https';
+import express from 'express';
+import { ApolloServer } from 'apollo-server-express';
+import { createServer } from 'graphql-ws';
+import { execute, subscribe } from 'graphql';
+import { schema } from 'my-graphql-schema';
+
+// create express
+const app = express();
+
+// create apollo server
+const apolloServer = new ApolloServer({
+    schema,
+    subscriptions: false, // disable subscriptions-transport-ws
```

Comment:
This is not needed because `(apollo-server-core) installSubscriptionHandlers` is only invoked if you're using `ApolloServer.listen`, Apollo's provided convenience if you don't create your own Express object.

Reply:
TBH, am not an Apollo expert... So I cannot confirm this, can you @she11sh0cked? [Seems like `ApolloServer` does something if the `subscriptions` is not `false`.](https://github.com/apollographql/apollo-server/blob/dfd87a0d836f96a5ad8e1538615056799d664f59/packages/apollo-server-core/src/ApolloServer.ts#L289-L323)

@kevinwei00 could you maybe point me to the location in code where this is checked?

Reply:
[In this function](https://github.com/apollographql/apollo-server/blob/main/packages/apollo-server-core/src/ApolloServer.ts#L599), the subscriptionServer is implemented by `subscriptions-transport-ws`. And [here](https://github.com/apollographql/apollo-server/blob/main/packages/apollo-server/src/index.ts#L119) is where it gets used (only in `ApolloServer.listen`).

Reply:
Gotcha! So only if you're calling the `listen` method on the apollo server the default `subscriptions-transport-ws` will kick in:
```
const apolloServer = new ApolloServer({ ... });
apolloServer.listen(...);
```

Reply:
Recipe updated in https://github.com/enisdenjo/graphql-ws/commit/b46a33c4c5ac060e192520399cd2c6afd30a927a.

Reply:
That's my understanding, yes.

Reply:
Thanks @kevinwei00, I love simplifications! 😄

Reply:
Should we note that in the recipe? Could lead to confusion if someone uses `apolloServer.listen(...);`, right?

Reply:
I wouldn't, the recipe explicitly uses the http server. They should be as simple as possible, the MVP for usage.